include/common/scaled-rect-buffer.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_SCALED_RECT_BUFFER_H
+#define LABWC_SCALED_RECT_BUFFER_H
+
+#include <stdint.h>
+
+struct wlr_scene_tree;
+struct wlr_scene_buffer;
+struct scaled_scene_buffer;
+
+struct scaled_rect_buffer {
+	struct wlr_scene_buffer *scene_buffer;
+	struct scaled_scene_buffer *scaled_buffer;
+	int width;
+	int height;
+	int border_width;
+	float fill_color[4];
+	float border_color[4];
+};
+
+/**
```

Comment:
We've not been super strict on this, but when using `/**` I prefer that we stick to kernel doc format.

- https://github.com/labwc/labwc/blob/master/CONTRIBUTING.md#linux-kernel-style-basics
- https://docs.kernel.org/doc-guide/kernel-doc.html

Reply:
I moved `TODO: ...` to `scaled_rect_buffer_create()` so that it doesn't look like the description of a parameter.
I didn't add descriptions of every parameter since they're self-explanatory.